docs/content/direct/its.md

```diff
@@ -1,10 +1,22 @@
-# Inventory and Transport Spaces
+# Inventory and Transport Spaces (ITS)
 
-This document will tell users what they need to know about these.
+An Inventory and Transport Space (ITS) is a special type of space in KubeStellar that manages the inventory of clusters (Workload Execution Clusters, or WECs) and handles the transport of workloads to those clusters. The ITS is typically implemented as a KubeFlex ControlPlane of type `vcluster` with Open Cluster Management (OCM) installed.
 
-Including the fact that the KubeFlex hosting cluster can play the role
-of ITS, and how to make that happen.
+## Role of ITS
+- Maintains the inventory of WECs using OCM's ManagedCluster objects.
+- Acts as the hub for distributing workloads to registered WECs.
+- Hosts mailbox namespaces for each WEC, where workload bundles (ManifestWork objects) are delivered.
 
-Include the fact that "Create an ITS" includes installing the relevant OCM machinery in it, but the step of registering it as a KubeFlex control plane is a separate step.
+## Creating an ITS
+1. **Create a ControlPlane for ITS**: Use KubeFlex to create a new ControlPlane of type `vcluster` and run the OCM post-create hook to install OCM components.
+   ```shell
+   kflex create its1 --type vcluster -p ocm
```

Comment:
You mean use the PostCreateHook named "ocm" defined in the KubeFlex repo? There is more to a KubeStellar ITS. Look at what the core Helm chart does.